docs/deploy-with-docker-on-linuxone.md

```diff
@@ -24,150 +24,39 @@ We will be using a Ret Hat base image for this journey, so be sure to chose the
 
 ### 1. Setup
 
-First, let's create a directory for our scenario:
+First, install docker and docker-compose from the instructions in the LinuxONE
+[repo](https://github.com/IBM/Cloud-Native-Workloads-on-LinuxONE)
 
-```text
+Then let's create a directory for our scenario:
+
+```shell
 $ mkdir gitlabexercise
 $ cd gitlabexercise
 ```
 
-You will also need a `requirements.txt` file in the directory with the contents:
-
-```text
-gitlab
-postgres
-redis
-```
-
-### 2. Create Dockerfiles
+### 2. Copy scripts and  Dockerfiles
 
-Next we need to write a few Dockerfiles to build our Docker images.  In the
-project directory, create the following three files with their respective
-content:
+Copy the files found in the `linuxone` folder:
```

Comment:
Hmm I was confused a bit at here, maybe the user can just `git clone` and get the files or give some `scp` instructions for copying the files to LineOne machine.